src/CRTM_Forward_Module.f90

```diff
@@ -489,6 +489,9 @@ FUNCTION profile_solution (m, Opt, AncillaryInput) RESULT( Error_Status )
          IF( Opt%n_Stokes > 0 ) THEN
             RTV(:)%n_Stokes = Opt%n_Stokes
             RTV_Clear(:)%n_Stokes = Opt%n_Stokes
+         ELSE
+            RTV(:)%n_Stokes = 0
```

Comment:
I think this `ELSE statement` is unnecessary, because Opt%n_Stokes is by default set as 1 (See `RTSolution/RTV_Define.f90`)  that `Opt%n_Stokes > 0` is always true, unless users accidentally set` Opt%n_Stokes <= 0`, which is wrong.

I feel this `IF statement` is here to prevent users from assigning a wrong number to `Opt%n_Stokes`, and to make sure it is at least set as 1 in CRTM.

A lot of the conditional calculations are based on `RTV%n_Stokes`, especially `IF %n_Stokes  == 1`, so it should never be set as 0.